src/main/scala/nl.knaw.dans.easy.validatebag/EasyValidateDansBagServlet.scala

```diff
@@ -44,7 +44,9 @@ class EasyValidateDansBagServlet(app: EasyValidateDansBagApp) extends ScalatraSe
     } yield Ok(body)
 
     result.getOrRecover {
-      case t: IllegalArgumentException => BadRequest(s"Input error: ${ t.getMessage }")
+      case t: IllegalArgumentException =>
+        logger.warn(s"validation of bag was unsuccessful with message: ${ t.getMessage } ")
```

Comment:
Could you mention which bag was unsuccessful? not sure from which places this exception can come from, but it would be useful to have some more info on which bag validation was unsuccessful.

(besides, given that an invalid bag also produces a `OK_200`, does this exception really say that bag validation was unsuccessful?)

Reply:
I guess I should rephrase that sentence. It's an Illegal argument exception (user error) so I think we could not resolve the bag name (invalid / absent uri or something similar). Would it be better log something like: validation process failed due to : e.getMessage??